src/components/hv-picker-field/index.ios.tsx

```diff
@@ -37,50 +37,51 @@ export default class HvPickerField extends PureComponent<HvComponentProps> {
   getPickerInitialValue = (): string => {
     const value = this.getValue();
     const pickerItems: Element[] = this.getPickerItems();
-    if (
-      pickerItems
-        .map((item: Element) => item.getAttribute('value'))
-        .includes(value)
-    ) {
+    const valueExists = pickerItems.some(
+      item => item.getAttribute('value') === value,
+    );
+    if (valueExists) {
       return value;
     }
-    if (pickerItems.length > 0) {
-      return pickerItems[0].getAttribute('value') || '';
-    }
-    return '';
+    return pickerItems.length > 0
+      ? pickerItems[0].getAttribute('value') || ''
+      : '';
```

Comment:
@Shaykoo your branch needs a rebase with the latest master, this is your previous change that got merged in #954